PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/DeviceLoginWindow.java

```diff
@@ -82,6 +84,14 @@ public DeviceLoginWindow(final AuthenticationContext ctx, final DeviceCode devic
                 }
             }
         });
+        // Apply JLabel's font to JEditorPane
+        final Font font = UIManager.getFont("Label.font");
+        if (font != null) {
+            final String cssRule = String.format("body { font-family: %s ; font-size: %s pt ;}", font.getFamily(),
+                font.getSize());
+            ((HTMLDocument) editorPanel.getDocument()).getStyleSheet().addRule(cssRule);
```

Comment:
I don't see any reason to using css rules here, what do you think of adding in-place css insder createHtmlFormatMessage?

Reply:
Changed to use JEditorPane.HONOR_DISPLAY_PROPERTIES, which will use component setting when font or foreground color is not specified in the styled text.